packages/ui/src/components/alert/alert.types.ts

```diff
@@ -27,6 +27,11 @@ export type AlertProps = {
    * The alert icon is automatically rendered based on look. The icon can be overriden via this prop, for info look alerts only.
    */
   icon?: React.ElementType;
+  /**
+   * The size of the alert icon.
+   * @default undefined
+   */
+  iconSize?: 'xsmall' | 'small' | 'medium' | 'large' | 'flex';
```

Comment:
Remove flex option. Double check with Kate if we should allow 'xsmall' and 'xlarge' for icons since ticket says we should only allow sm,md and lg options